### ISSUE TITLE
Unable to Adjust the Order of Merged Comics

### ISSUE THREAD

**Issue by RejectVanity** [komikku-app/komikku#367]
### Steps to reproduce

1. Merge any comic books,
2. Adjust the order of merged comics in 'Merge settings'.
![Screenshot_2024-09-17-17-11-21-803_app komikku beta-edit](https://github.com/user-attachments/assets/a7dbdc7f-8d86-450a-abf4-27964f1d4d79)

### Expected behavior

The order of merged comics can be adjusted normally.

### Actual behavior

Unable to adjust.

### Komikku version

1.11.3-9318

### Android version

Android 14

### Device

POCOF4 (munch)

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open or closed issue.
- [X] I have written a short but informative title.
- [X] If this is an issue with an official extension, I should be opening an issue in the [extensions repository](https://github.com/komikku-app/extensions/issues/new/choose).
- [X] I have gone through the [FAQ](https://mihon.app/docs/faq/general) and [troubleshooting guide](https:/mihon.app/docs/guides/troubleshooting/).
- [X] I have updated the app to version **[1.11.2](https://github.com/komikku-app/komikku/releases/latest)**.
- [X] I have updated all installed extensions.
- [X] I will fill out all of the requested information in this form.

**Comment by cuong-tran** [comment]
Did it ever work before?

**Comment by RejectVanity** [comment]
> Did it ever work before?

I just reverted to the 'Komikku-arm64-v8a-r9312.apk' version, and I am unable to adjust the order.

**Comment by cuong-tran** [comment]
There has never been any ability to adjust merged entries order yet. Why do you need to adjust order?

**Comment by RejectVanity** [comment]
> There has never been any ability to adjust merged entries order yet. Why do you need to adjust order?

The symbol in front that looks like ‘=’ generally indicates that you can press and hold it to adjust the order.
I usually use the ‘merge’ function to combine serialized manga, either by parts (such as Part 1, Part 2) or by seasons (such as Season 1, Season 2).

**Comment by JoeloJoestar** [comment]
The order matters for the "Dedupe by priority" mode, and so it lets you set up the "priority". I just checked this and it is orderable only after selecting the "Dedupe by priority" mode and not the others. It is not very intuitive because there are no visual clues of this activating option, so this could be improved. The function though works, as in latest preview build.

**Comment by RejectVanity** [comment]
> There has never been any ability to adjust merged entries order yet. Why do you need to adjust order?

When adjusting the order of comics, the chapters are rearranged as well. For example, the first and second chapters of comic "a" and the first and second chapters of comic "b" are originally ordered as a1, a2, b1, b2. After the order is adjusted, they become b1, b2, a1, a2.

**Comment by JoeloJoestar** [comment]
@RejectVanity please read my comment above if you did not and you find out it is working.

**Comment by RejectVanity** [comment]
> @RejectVanity please read my comment above if you did not and you find out it is working.

That is only to adjust the priority of removing duplicates, not to adjust the order of comics (in other words, chapters).

**Comment by JoeloJoestar** [comment]
@RejectVanity as I understand the Merge function has been designed to fuse sources together to fill the missing chapters void, and so it makes to have this option for deduplication, why would you want the same chapters of more sources stacked like you described instead of just having multiple entries?

**Comment by JoeloJoestar** [comment]
It's like doubling the number of chapters for a Manga.. Imagine to track this entry.. It's also against every sorting logic currently implemented.

**Comment by RejectVanity** [comment]
> It's like doubling the number of chapters for a Manga.. Imagine to track this entry.. It's also against every sorting logic currently implemented.

My idea is to merge the entire series into one book (e.g., Season 1, Season 2...), as some works are serialized in parts. I believe this should be considered a feature request, not a "bug" report.

**Comment by RejectVanity** [comment]
> It's like doubling the number of chapters for a Manga.. Imagine to track this entry.. It's also against every sorting logic currently implemented.

Here is the translation:

In the case where 'Allows deduplication' is disabled, I would like to allow adjusting the order of the comics as I mentioned.

**Comment by JoeloJoestar** [comment]
Alright @RejectVanity I get you and I agree. The Merge function can be even more useful after this, so the point for merging won't be to just fill the voids but to merge more separated chapters of a series into one, then making the library less disordered.

So like I said, the current issue is that only if we activate the deduplication we can adjust the order of the chapters somehow, so not yet possible to have a1, a2, b1, b2 if we don't wanna deduplicate.

Besides, it would be very useful too if we could just drag-and-drop single chapters of an entry.
